src/signal.h

```diff
@@ -20,29 +21,37 @@
 #define CODE_GPS_L1CA_SUPPORT     1
 #define CODE_GPS_L2CM_SUPPORT     0
 #define CODE_SBAS_L1CA_SUPPORT    1
+#define CODE_GLO_L1CA_SUPPORT     0
+#define CODE_GLO_L2CA_SUPPORT     0
 /* =================================== */
 
 /** \addtogroup signal
  * \{ */
 
 /* Number of signals on each code supported
  * on the current hardware platform. */
-#define PLATFORM_SIGNAL_COUNT_GPS_L1CA      (CODE_GPS_L1CA_SUPPORT ?          \
```

Comment:
Yep, looking closer I noticed that Geany editor has the default line ending mark at column 72 for some reason... Sorry.